entries/y/ymobile.jp.json

```
@@ -0,0 +1,16 @@
+{
+  "Y!mobile": {
+    "domain": "ymobile.jp",
+    "contact": {
+      "facebook": "ymobile",
+      "twitter": "ymobileOfficial",
```

This seems to be the Twitter account used for support and feedback.
```suggestion
      "twitter": "ymobile_Care",
```